tasks/init.yaml

```diff
@@ -27,3 +30,13 @@ spec:
         else
           echo -n "false" > $(results.build.path)
         fi
+    - name: hacbs-init
+      image: registry.redhat.io/openshift4/ose-cli:v4.10
+      script: |
+        if [ "$(params.jvm-build-service-init)" == "true" ] && ! oc get configmap jvm-build-config &>/dev/null; then
+          oc create configmap jvm-build-config --from-literal enable-rebuilds="false"
```

Comment:
this configmap is also created in cases when I run `docker-build` or `nodejs-builder` pipelines, because that `jvm-build-service-init` parameter is added to those pipelines as well:
<img width="296" alt="Screenshot 2022-08-11 at 12 22 06" src="https://user-images.githubusercontent.com/4881144/184113651-fff9bd42-ef37-4903-9994-72618a4ae909.png">
I guess we need it only in `java-builder` pipeline?

Reply:
not for nodejs-builder, but for docker-build yes since there could be java build in dockerfile

Reply:
updated, so now the patch is applyed pipelines with label -> `pipelines.openshift.io/runtime == java` and `pipelines.openshift.io/runtime == generic`